gluecodium/src/main/java/com/here/gluecodium/cli/OptionReader.kt

```diff
@@ -176,10 +180,8 @@ object OptionReader {
     }
 
     fun printUsage() {
-        val header = "Gluecodium - Generate APIs from LimeIDL files\n\n"
-        val footer = "\nPlease report issues at /dev/null"
```

Comment:
Could have been replaced with a link to the actual issue tracker

Reply:
Done